src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -25,6 +24,28 @@ import {
   AVAILABLE,
 } from '../utils/appeals-v2-helpers';
 
+const appealsDownMessage = (
+  <div className="row" id="appealsDownMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
+        <h3>We’re sorry. Something went wrong on our end.</h3>
+        <p>Please refresh this page or try again later. If it still doesn’t work, you can call the Vets.gov Help Desk at <a href="tel:18555747286">1-855-574-7286</a> (TTY: <a href="tel:18008294833">1-800-829-4833</a>). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET).</p>
+      </div>
+    </div>
+  </div>
+);
+
+const recordsNotFoundMessage = (
+  <div className="row" id="recordsNotFoundMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
+        <h3>We’re sorry. We can’t find your records in our system.</h3>
+        <p>If you think they should be here, please try again later or call the Vets.gov Help Desk at <a href="tel:18555747286">1-855-574-7286</a> (TTY: <a href="tel:18008294833">1-800-829-4833</a>). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET).</p>
```

Comment:
It looks like this design has been updated from the guidance provided in the linked issue. @gnakm has signed off on the updated content, so it must be good to go.

Reply:
I must be missing something. What's the difference?